### DISCUSSION TITLE
Compilation for Adreno 888

### DISCUSSION THREAD

**Discussion by daviDarthemis** [ptitSeb/box64#1017] [General]
There are specific builds for the Adreno chip like 845 888, what exactly do they change?

**Comment by ptitSeb** [comment]
Compile flags, adapted to each cpu arch.
for 845: `-march=armv8.2-a+simd+crypto -mtune=cortex-a75.cortex-a55`
for 888: `-march=armv8.4-a+simd+crypto`

**Comment by daviDarthemis** [comment]
In the case of general systems arm, can I just use `arm64=on`?

**Reply by ptitSeb** [reply]
yes

**Comment by daviDarthemis** [comment]
In the case of the sd 855 I saw that it has the `cortex-a76` `cortex-a55`

**Reply by ptitSeb** [reply]
It's close enough to the 845, that small change wont make any significant performances change.

**Comment by daviDarthemis** [comment]
So would it be like this? `-march=armv8.2-a+simd+crypto -mtune=cortex-a76.cortex-a55`

**Reply by ptitSeb** [reply]
if you want.

**Comment by daviDarthemis** [comment]
Would it be possible to specific gpu data? Like clock shaders or something like that or it doesn't need this data because freedreno already looks for this

**Reply by ptitSeb** [reply]
no, box64 doesn't care about GPU. It just use OpenGL or Vulkan driver as-is without trying any control over it.
